release/karaf/itests/switchyard/pom.xml

```diff
@@ -300,4 +301,35 @@
             </plugin>
         </plugins>
     </build>
+
+  <profiles>
+        <profile>
+            <id>fuse-6.2.1</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <!-- 2.18.1 is failing with "When writing xml report stdout/stderr: /tmp/stdout (No such 
+                            file or directory)" -->
+                        <version>2.16</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <java.awt.headless>true</java.awt.headless>
+                                <drools.karaf.features.classifier>${drools.karaf.features.classifier}</drools.karaf.features.classifier>
+                            </systemPropertyVariables>
+                            <includes>
+                                <include>**/*Test*.java</include>
+                            </includes>
+                            <excludes>
+                                <exclude>**/Abstract*.java</exclude>
+                                <exclude>**/HelpdeskDemoQuickstartTest.java</exclude>
+                                <exclude>**/*$*</exclude>
+                            </excludes>
+                        </configuration>
+                    </plugin>
```

Comment:
There was a error on 6.2.1 libraries that made this itest to fail on plain karaf. That s why this itest is failing and was ignored before. 

This is fixed on fuse 6.3. 

Additionally I ensured that the Helpdesk feature is working on fuse 6.2.1.